cpp/src/arrow/adapters/orc/jni/CMakeLists.txt

```diff
@@ -0,0 +1,55 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
What do you think about centralizing Arrow JNI bindings into an cpp/src/arrow/jni folder? This may make the bindings easier to organize and develop

Reply:
Sure, I have no problem with that.